pkg/operator/controller.go

```diff
@@ -40,6 +42,9 @@ const (
 	kubeSystemNamespace      = "kube-system"
 	workqueueKey             = "changes"
 	defaultResyncDuration    = 10 * time.Minute
+	caBundlePath             = "/etc/pki/ca-trust/source"
+	caBundleFileName         = "ca-bundle.crt"
```

Comment:
Some churn here - the CA will now be mounted into `/etc/pki/ca-trust/extracted/tls-ca-bundle.pem`.

Reply:
One would assume that we will have an "index" called `tls-ca-bundle.pem` containing the file content inside the config map then. Right?

Reply:
I am going to solve this one as the current implementation does not contain this snippet anymore. Other than that we are not hard coding any paths on the new implementation.

Reply:
I guess i can resolve this comment as the original PR was completely refactored and we don't have these paths hardcoded anymore.